src/components/type.js

```diff
@@ -2,7 +2,7 @@ import { useEffect, useMemo, useRef, useState } from 'react';
 import styled from 'styled-components';
 
 const interval = 100;
-const idle = 1000;
+const idle = 1;
```

Comment:
I don't think we need to change this, this is an intentional design decision to leave the cursor visible for a second

Reply:
Oh I thought so, since the name was idle. I reverted the change to idle, tried to use the styled components, I think it should work fine now